.github/labeler.yml

```diff
@@ -0,0 +1,14 @@
+# Configuration for the Labeler action
```

Comment:
This didn't cover all of the ideas @compwron suggested in the original issue (#195) because the labels didn't exist. 

If anyone with label-creating powers is open to creating new issues, I can add more mappings here.

It looks like @compwron may have wanted more labels by file type:

>  erb/js/css, models, db/, and Pundit policy files

Reply:
Thanks! I added those 3 labels. I'll merge this and we'll... see what happens 😂